### PR TITLE
Code of conduct and governance documents

### PR DESCRIPTION
**Pull request by Pessimistress**

### CODE REVIEW

File: dev-docs/governance.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+## Committer
+
+### 2a. A committer's previledges
+
+- Roadmap planning
+  + Submit, approve and comment on RFCs.
+  + Participate in the planning meeting at the beginning of each release cycle (3a).
+  + Join the developer channel for the ongoing technical discussions, development activities and release updates.
+- Write permission to the repo, and membership to the NPM organization
```
Comment (ibgreen) [inline dev-docs/governance.md]: Should there be a section on "Steering Committee Members Privileges", in particular regarding Admin permissions to the repos etc? 

I assume all Steering Committee Member should be e.g. github admins - if not them, then who?

File: dev-docs/governance.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+An existing committer can be nominated to join the steering committee by other steering committee members. The nomination is formally submitted as a PR to update the contribution guide. Nominations are approved by the steering committee.
+
+###  1c. Leaving the steering committee
+
+If a steering committee member can/will no longer perform their duties, they can be removed from the steering committee by the consensus of the rest of the steering committee members.
+
+
+## Committer
+
+### 2a. A committer's previledges
```
Comment (ibgreen) [inline dev-docs/governance.md]: Typo "privileges".

General tip: I'd paste this text into google doc or use some other means to get it spell checked.

File: dev-docs/governance.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+- Agrees to dedicate time to perform their committee duties.
+
+An existing committer can be nominated to join the steering committee by other steering committee members. The nomination is formally submitted as a PR to update the contribution guide. Nominations are approved by the steering committee.
+
+###  1c. Leaving the steering committee
+
+If a steering committee member can/will no longer perform their duties, they can be removed from the steering committee by the consensus of the rest of the steering committee members.
+
+
+## Committer
```
Comment (ibgreen) [inline dev-docs/governance.md]: The term "committer" is potentially ambiguous as a lot of people make commits without actually getting e.g. write privileges to the repo. 

One way out would be to use a qualifying word, e.g. "Active Committer", or "Registered Committer", etc.

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
+
```
Comment (ibgreen) [inline CODE_OF_CONDUCT.md]: Empty line?

File: dev-docs/governance.md (hunk 3 above)
Comment (Pessimistress) [reply dev-docs/governance.md]: This term is suggested by the Uber OSS office.

**Comment by Pessimistress** [comment]
@ibgreen If we do move to our own organization, then I suggest we make a dedicated repo for dev process/governance docs, which will apply to all the repos under the org.

**Pull request merged**